app/assets/javascripts/components/flexible_roster.jsx

```diff
@@ -0,0 +1,158 @@
+import SortHelpers from '../helpers/sort_helpers.jsx';
```

Comment:
This is the FlexibleRoster component. Reuses a bunch of code from students_table.jsx